pre_commit_hooks/file_contents_sorter.py

```diff
@@ -65,6 +65,11 @@ def main(argv: Sequence[str] | None = None) -> int:
     )
     args = parser.parse_args(argv)
 
+    if args.ignore_case and args.unique:
+        print('ERROR: usage of --unique and --ignore-case is unsupported. \
+        Please update your configuration.')
+        return FAIL
```

Comment:
don't use backslashes like this -- also this is going to render very strangely

also this behaviour needs to be tested (`capsys` might help) -- but then again you can probably just make this use argparse's mutually exclusive options